docs/latex/Inhalt/literatur.bib

```diff
@@ -0,0 +1,7 @@
+@book{Martin.2018,
+  title={Clean architecture: a craftsman's guide to software structure and design},
+  author={Martin, Robert C and Grenning, James and Brown, Simon and Henney, Kevlin and Gorman, Jason},
+  number={s 31},
```

Comment:
I can't find issue "s 31" of this book. See http://bib-it.sourceforge.net/help/fieldsAndEntryTypes.php#number

Reply:
Hmmm, das war von Google Scholar generiert. Ich ändere das.